<launch>







<!-- send image to snapshotter -->

<node pkg="cv_mech_turk" type="snapper.py" name="snap" output="screen">

   <param name="session"  value="pre-test-w-env-L1-s" />
   <param name="frame"  value="stereo_l_stereo_camera_frame" />
   <param name="img_dir"  value="/tmp/submit_images" />

   <param name="block_submit"  value="1" />

   <remap from="image"  to="/hist/stereo/left/image_rect" />

</node>





</launch>
